package-res/resources/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -213,6 +213,10 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
             return true;
           }
 
+          if(dateFormat.match(/mm/)) {
+            return true;
+          }
+
```

Comment:
That means we're never showing time (which is probably reasonable on our current date picker). ok, then.